modules/dxp/apps/analytics/analytics-reports-document-impl/bnd.bnd

```diff
@@ -0,0 +1,3 @@
+Bundle-Name: Liferay Analytics Reports Document Implementation
```

Comment:
The module should be named `Liferay Analytics Reports Document Library Implementation`. The folder should be `analytics-reports-document-library-impl` and the packages `...document.library...`.

This is because we use the same names from the main folder of the application we are adding support for. In this case `document-library`.

Reply:
Done!
Thank you for the explanation